packages/react-native-win32/docs/api/react-native-win32.accessibilitystates.md

```diff
@@ -1,11 +0,0 @@
-<!-- Do not edit this file. It is automatically generated by API Documenter. -->
-
-[Home](./index.md) &gt; [@office-iss/react-native-win32](./react-native-win32.md) &gt; [AccessibilityStates](./react-native-win32.accessibilitystates.md)
-
-## AccessibilityStates type
```

Comment:
do we have a mechanism to mark a type as deprecated/removed in the docs?

Reply:
`accessibilityStates` was deprecated in upstream RN 0.61, then removed in 0.62. This is a Win32 specific fork of the property to add more to it. It wasn't maintained for 0.61, so we're removing it in alignment with upstream 0.62. We explicitly want to cause a break here to prevent regressions down the line. Forwarded you a thread with extra context offline.